README.md

```diff
@@ -85,7 +85,20 @@ follow these instructions:
 2. Open the project's [Global variables](https://docs.altostra.com/cloud-designer/global-environment).
 3. Change the value of the variable named `GPT_API_KEY_PARAM`.
 4. Click on Save.
-5. Click **on the icon** of the `api-key-params` SSM Parameters resource.
+5. Click on the `api-key-params` SSM Parameters resource.
```

Comment:
I phrased and emphasized this for a reason

https://github.com/Altostra-Labs/altostra-template-gpt-web-app/assets/57314279/b35b8d9f-3b20-4572-88f2-5e4810be6c7a